src/main/java/dal/fatih/todoly/Task.java

```diff
@@ -1,32 +1,31 @@
 package dal.fatih.todoly;
 
 import java.util.Date;
-import java.util.concurrent.atomic.AtomicInteger;
+import java.util.UUID;
 
 public class Task {
 
-    private String id;
+    private UUID id;

```

Comment:
It is good that you discovered UUID by yourself. This is the beet way to handle unique IDs in programming

Reply:
Thnx.